src/h3lib/lib/baseCells.c

```diff
@@ -886,3 +887,18 @@ Direction _getBaseCellDirection(int originBaseCell, int neighboringBaseCell) {
     }
     return INVALID_DIGIT;
 }
+
+/**
+ * getBaseCells generates all base cells storing them into the provided
+ * memory pointer.
+ *
+ * @param out H3Index* the memory to store the resulting base cells in
```

Comment:
We should document how large this buffer needs to be - if NUM_BASE_CELLS is needed, that perhaps should be in h3api.h.

Reply:
I'm not sure the JS build can currently access constants, even if they're in `h3api.h`. I think the version that would be most consistent with other pass-in-allocated-memory functions would be a new function `res0IndexCount` or similar, which would return `NUM_BASE_CELLS`.